docs/index.rst

```diff
@@ -53,6 +53,23 @@ Quickstart
     print("Testing Accuracy: {:.3f} %".format(acc))
     >>> Testing Accuracy: 98.667 %
 
+.. code-block:: python
+
+    from sklearn.datasets import load_boston
+    from sklearn.model_selection import train_test_split
+    from sklearn.metrics import mean_squared_error
+
+    from deepforest import CascadeForestRegressor
+
+    X, y = load_boston(return_X_y=True)
+    X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=1)
+    model = CascadeForestRegressor(random_state=1)
+    model.fit(X_train, y_train)
+    y_pred = model.predict(X_test)
+    acc = mean_squared_error(y_test, y_pred)
```

Comment:
changing the variable name from `acc` to `mse` would be better ;-)

Reply:
Realise the print statement variable also needs to change, will fix this tomorrow